web/src/components/CodeBlockWithTitle.js

```diff
@@ -1,12 +1,12 @@
 import React, { useState } from 'react'
 import CodeBlock from '@theme/CodeBlock'
 
-export default function CodeBlockWithTitle ({ title, children, language, metastring }) {
+export default function CodeBlockWithTitle ({ title, children, language }) {
   return (
     <div className='code-with-header'>
       <div className="code-header">{ title }</div>
 
-      <CodeBlock className={language} metastring={metastring}>
```

Comment:
The syntax for this changed, and I removed `metastring` since we did not use it, and it was mentioned to avoid: https://docusaurus.io/docs/next/markdown-features/code-blocks#usage-in-jsx

Reply:
nice! thanks a lot for updating this.